base_kanban_stage/tests/test_base_kanban_abstract.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2016-2017 LasLabs Inc.
+# Copyright 2019 ForgeFlow.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from collections import deque
+
+from odoo import api, models
+from odoo.tests.common import TransactionCase
+
+
+class BaseKanbanAbstractTester(models.AbstractModel):
+    _name = "base.kanban.abstract.tester"
+    _description = "base kanban abstract tester"
+    _inherit = "base.kanban.abstract"
+
+
+class TestBaseKanbanAbstract(TransactionCase):
+    def _init_test_model(self, model_cls):
+        model_cls._build_model(self.registry, self.cr)
+        model = self.env[model_cls._name]
+        # setup_models():
+        model._prepare_setup()
+        model._setup_base()
+        model._setup_fields()
+        model._setup_complete()
+        # init_models():
+        self.registry._post_init_queue = deque()
+        self.registry._foreign_keys = {}
+        model._auto_init()
+        model.init()
+        self.env["ir.model"]._reflect_models([model._name])
+        self.env["ir.model.fields"]._reflect_fields([model._name])
+        self.env["ir.model.fields.selection"]._reflect_selections([model._name])
+        self.env["ir.model.constraint"]._reflect_constraints([model._name])
+        while self.registry._post_init_queue:
+            func = self.registry._post_init_queue.popleft()
+            func()
+
```

Comment:
> 

Hello, 
I'm facing some error like this,

  File "/home/odoo/workspace/community/odoo_14.0/odoo/sql_db.py", line 100, in check
    raise psycopg2.OperationalError('Unable to use a closed cursor.')
so.............
My suggestion is it exactly resolves as  what I did before

Reply:
you are facing that error because test is broken. put `return model`, then the test should work and you won't see that error. Also, please mark as resolved all my comments